test/unit/crawlerTests.js

```diff
@@ -545,6 +545,7 @@ describe('Crawler convert to document', () => {
       expect(metadata.etag).to.be.equal(42);
       expect(metadata.links).to.be.not.null;
       expect(metadata.fetchedAt).to.be.not.null;
+      expect(metadata.rootRequest).to.be.undefined;
```

Comment:
Interesting, so undefined is different than null?

Reply:
> Interesting, so undefined is different than null?

It is.